pkgs/by-name/ma/martian-grotesk/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "martian-grotesk";
+  version = "1.0.0";
+
+  src = fetchzip {
+    url = "https://github.com/evilmartians/grotesk/releases/download/v${version}/martian-grotesk-${version}-otf.zip";
+    sha256 = "sha256-rYOSvjKjO7Lc84n7WXKPfwkuTSG2fAFU1c3hSeCXD7o=";
+    stripRoot = false;
+  };
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  doCheck = false;
+  dontFixup = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 -t $out/share/fonts/opentype/ ./otf/*.otf
+
+    runHook postInstall
+  '';
+
```

One more suggestion, this one would automatically create a pull request when there is an update. (You'd also need to add nix-update-script at the top.)

```suggestion

passthru.updateScript = nix-update-script { };
```